### PR TITLE
Add content for api response on upload SDK methods

### PR DESCRIPTION
**Pull request by jace-roell**
**What It Does**
Add content for api response on upload SDK methods

**How to Test**
Invoke one of the following SDK methods and view results.
- bufferToUss
- streamtoUss
- bufferToDataSet
- streamToDataSet

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [x] added/updated automated tests
- [x] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Review by anaxceron — APPROVED** [review]

**Review by traeok — APPROVED** [review]
Thanks Jace for the enhancement - this LGTM!

I was curious to get your thoughts on setting the `from` property in `apiResponse` to include the bytes when its a buffer? You can use the `Buffer.prototype.inspect` or ` function to do so:

```ts
const buf = Buffer.from("test string").inspect();
// output: '<Buffer 74 65 73 74 20 73 74 72 69 6e 67>'
```

If we only want to show the first _x_ bytes to prevent a massive string, you can use the `subarray(0, x)` function on the buffer before calling `inspect`. It's definitely not required, but I figured it was worth mentioning since the hardcoded strings `Buffer<>` and `Stream<>` appear to be inspired by this output.

File: packages/zosfiles/src/methods/upload/Upload.ts [hunk 1]
```diff
@@ -242,7 +246,11 @@ export class Upload {
         const uploadRequest: IRestClientResponse = await ZosmfRestClient.putExpectFullResponse(session, requestOptions);
 
         // By default, apiResponse is empty when uploading
-        const apiResponse: any = {};
+        const apiResponse: any = {
+            success: true,
+            from: "Stream<>",
```
Comment (zFernand0) [inline packages/zosfiles/src/methods/upload/Upload.ts:251]: Nitpicky comment...
Can we surround the word Steam (or buffer) in the angle brackets? 
`<Stream>` or `<Buffer>`

File: packages/zosfiles/src/methods/upload/Upload.ts [hunk 2]
```diff
@@ -184,7 +184,11 @@ export class Upload {
         const uploadRequest: IRestClientResponse = await ZosmfRestClient.putExpectFullResponse(session, requestOptions);
 
         // By default, apiResponse is empty when uploading
-        const apiResponse: any = {};
+        const apiResponse: any = {
+            success: true,
+            from: "Buffer<>",
```
Comment (zFernand0) [inline packages/zosfiles/src/methods/upload/Upload.ts:189]: I like Trae's idea for buffers

And in theory we could do the same for streams as long as we allow the `from` property to be something other than a string...

we could do an `util.inspect` and get the following output for stream...
For example: `util.inspect(readableStream, { showHidden: false, depth: -1}));`
```shell
zFernand0 @ Mac (arm64) ~/gh/zowe/cli.master                                                                       [13:50:28]
> $ node test.mjs                                                                           ⬡ system [±api-response-fix ●(✹)]
[ReadStream] # This is with depth -1
                                                                                                                              
zFernand0 @ Mac (arm64) ~/gh/zowe/cli.master                                                                       [13:50:58]
> $ node test.mjs                                                                           ⬡ system [±api-response-fix ●(✹)]
ReadStream { # This is with depth 0
  fd: null,
  path: 'package.json',
  flags: 'r',
  mode: 438,
  start: undefined,
  end: Infinity,
  pos: undefined,
  bytesRead: 0,
  _events: [Object],
  _readableState: [ReadableState],
  _maxListeners: undefined,
  _eventsCount: 1,
  [Symbol(kFs)]: [Object],
  [Symbol(kIsPerformingIO)]: false,
  [Symbol(shapeMode)]: true,
  [Symbol(kCapture)]: false
}
```

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋 

left a few minor comments, but the code (and tests LGTM!) 🙏

**Review by gejohnston — APPROVED** [review]

File: packages/zosfiles/__tests__/__unit__/methods/upload/Upload.unit.test.ts [hunk 3]
```diff
@@ -22,7 +22,7 @@
 import { Upload } from "../../../../src/methods/upload/Upload";
 import { List } from "../../../../src/methods/list/List";
 import { Utilities } from "../../../../src/methods/utilities/Utilities";
-
+import { inspect } from "util";
```
[the remark this answers is not in the document]
Comment (jace-roell) [reply packages/zosfiles/__tests__/__unit__/methods/upload/Upload.unit.test.ts:25]: resolved in latest PR 👍

**Review by awharn — APPROVED** [review]

**Pull request merged**